### PR TITLE
PR for getting e2e deployment working [WIP]

### PR DESCRIPTION
**Pull request by jainr**
This PR addresses following things to get the e2e deployment working after earlier PRs for individual pieces were merged.
- Docker UI Image registry now points to a non-personal repo
- Adds connection string for SQL to UI App Settings
- Cleans up some of the excess parameters that were requested as part of SQL server creation

TODO
SQL extension for bacpac import expects the storage account key to be passed even though the bacpac URL is a public blob, we need to brainstrom this and come up with another design.
Add Bacpac doc from @windoze after we decide on the approach to best address this.

**Nice to have**
Test whether AAD app could replace Service Principal ID. (not a blocker and could be addressed later on as an enhancement)

### CODE REVIEW

**Review by xiaoyongzhu — APPROVED** [review]

**Comment by xiaoyongzhu** [comment]
LGTM

**Review by Yuqing-cat — APPROVED** [review]
We could have a follow up PR to add some screenshot guide and FAQ for better deployment experience.

File: docs/how-to-guides/azure_resource_provision.json [hunk 1]
```diff
@@ -18,57 +18,45 @@
...
       "metadata": {
         "description": "Specifies the username for admin"
       }
     },
-    "adminPassword": {
-      "type": "securestring",
+    "sqlAdminPassword": {
+      "type": "SecureString",
       "metadata": {
         "description": "Specifies the password for admin"
```
Comment (blrchen) [inline docs/how-to-guides/azure_resource_provision.json:61]: >Specifies the password for admin [](http://example.com/codeflow?start=24&length=32)

Enya tried a deployment and she thought this one is for setup admin username and password for feathr. Can we make description more meaningful so user know this is for sql admin?

Comment (jainr) [reply docs/how-to-guides/azure_resource_provision.json:61]: I will make it more clear in upcoming PRs.

**Pull request merged**